app/js/helper_profile_controller.js

```diff
@@ -0,0 +1,74 @@
+'use strict';
+
+/* global SumoDB, nunjucks  */
+
+(function(exports) {
+  var MSG_NO_QUESTIONS = 'No questions found';
+  var PROFILE_DETAILS_TMPL = 'helper_profile.html';
+  var QUESTION_LIST_TMPL = 'question_list_day.html';
+
+  var username;
+  var headline;
+  var profile_details_container;
+  var questions_container;
+
+  function load_profile() {
+    var promise = SumoDB.get_user(username, false, 96);
+
+    promise.then(function(user) {
+      headline.innerHTML = user.display_name;
+
+      var profile_html = nunjucks.render(PROFILE_DETAILS_TMPL, {
+        user: user
+      });
+
+      profile_details_container.innerHTML = profile_html;
+
+      SumoDB.get_my_questions(user).then(function(response) {
```

Comment:
get_my_questions returns questions you have created, not the ones you have solved. We need a different API call.